<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qu·∫£n L√Ω Task ƒê∆°n Gi·∫£n</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_tieu_de">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>font-size: 18px; font-weight: bold; color: blue;</string>
    </property>
    <property name="text">
     <string>üìù QU·∫¢N L√ù TASK ƒê∆†N GI·∫¢N</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_user">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>15</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>font-size: 14px; color: green;</string>
    </property>
    <property name="text">
     <string>Xin ch√†o: User</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dang_xuat">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: red; color: white; font-weight: bold;</string>
    </property>
    <property name="text">
     <string>ƒêƒÉng xu·∫•t</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ds_task">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>font-size: 14px; font-weight: bold;</string>
    </property>
    <property name="text">
     <string>üìã Danh s√°ch Task:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_task">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>450</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>font-size: 14px; border: 2px solid gray;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_them_task">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: green; color: white; font-size: 14px; font-weight: bold;</string>
    </property>
    <property name="text">
     <string>‚ûï Th√™m Task</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sua_task">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>170</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: orange; color: white; font-size: 14px; font-weight: bold;</string>
    </property>
    <property name="text">
     <string>‚úèÔ∏è S·ª≠a Task</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_xoa_task">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>230</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: red; color: white; font-size: 14px; font-weight: bold;</string>
    </property>
    <property name="text">
     <string>üóëÔ∏è X√≥a Task</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_doi_trang_thai">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: blue; color: white; font-size: 14px; font-weight: bold;</string>
    </property>
    <property name="text">
     <string>üîÑ ƒê·ªïi Tr·∫°ng Th√°i</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
